chainguard-dev/edu
# Adding new Security Advisories content to the Academy

## Type of change
New documentation

### What should this PR do?
This PR adds two new items to the Academy: An article outlining *how* Chainguard issues security advisories (i.e. the stages that lead up to the advisory's addition to the database, and how the status labels are chosen and updated as more information is revealed), and a guide explaining how users can use the `wolfictl` command line tool to view, search, create, and update security advisories.
I also opted to move the existing security advisories article (regarding how to use the advisories feed to inform your next steps) into a new bucket simply titled "Security Advisories" so that all three of these documents share the same home.

Resolves https://github.com/chainguard-dev/internal/issues/3303 
Resolves https://github.com/chainguard-dev/internal/issues/4061

### Why are we making this change?
Customers have expressed confusion with the security advisories page. Namely, customers want to understand how the security advisories page is created. What factors impact the label chosen for the advisory? Our existing documentation on this topic helps users understand the meaning of the advisories, but this new documentation should clarify any lingering questions on how decisions are made under-the-hood. Particularly, questions surrounding "why does my scanner show this and your feed doesn't?" should be answered, as the article walks through the step-by-step process from vulnerability disclosure to advisory creation and remediation. Those curious about the tooling used to issue an advisory can follow the guide using wolfictl to learn how they could do this themselves.

### What are the acceptance criteria? 
This set of articles should meet the goal set out by both the above explanation and referenced issues. They should be clearly written, technically accurate, and comprehensive on the topic so customers walk away with a better understanding of the security advisories feed. 

### How should this PR be tested?
Please check out the deploy preview or build the site locally since I've added some changes to the directory hierarchy of some of the articles. Checking style, links, and technical accuracy is also much appreciated.
